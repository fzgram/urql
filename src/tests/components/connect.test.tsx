/* tslint:disable */

import React from 'react';
import Connect from '../../components/connect';
import renderer from 'react-test-renderer';

describe('Client Component', () => {
  it('should provide a context consumer and pass through to client', () => {
    // @ts-ignore
    const component = renderer.create(
      <Connect children={args => <div {...args as any} />} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
