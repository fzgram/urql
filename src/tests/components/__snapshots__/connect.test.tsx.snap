// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Client Component should provide a context consumer and pass through to client 1`] = `
<div
  cache={
    Object {
      "invalidate": [Function],
      "invalidateAll": [Function],
      "read": [Function],
      "update": [Function],
    }
  }
  client={Object {}}
  data={null}
  error={null}
  fetching={false}
  loaded={false}
  refetch={[Function]}
  refreshAllFromCache={[Function]}
/>
`;
